src/activity_actor.cpp

```diff
@@ -7598,6 +7599,96 @@ std::unique_ptr<activity_actor> vehicle_unfolding_activity_actor::deserialize( J
     return actor.clone();
 }
 
+int heat_activity_actor::get_aviliable_heater( Character &p, item_location &loc ) const
+{
+    int available_heater = 1;
```

Comment:
If there's no heater, no energy.
```suggestion
    int available_heater = 0;
```